README.md

```diff
@@ -194,6 +194,16 @@ Examples
 
 Both `--exclude` and `--include` options can be used together but `--exclude` takes precedence. You can specify the `--exclude` and `--include` options multiple times to allow for multiple filter expressions.
 
+##### Note for Powershell / VSTS users
```

Comment:
This should be below the `MSBuild` section

Reply:
thanx, did the modification